examples/buildscripts/checkstyle.xml

```diff
@@ -0,0 +1,233 @@
+<?xml version="1.0"?>
```

Comment:
Can we share this file between the root project and examples? Same for license.

Reply:
Sure.

The reason I copied that file here is to make examples "standalone" without referencing files outside the folder. The examples is still a sub-folder, not in a separate project so I think it should be fine to reuse those configurations.

Reply:
+1 to reuse.